test/manual/message.cc

```diff
@@ -190,7 +190,7 @@ int main(int ac, char ** av) {
             locator::token_metadata::config tm_cfg;
             tm_cfg.topo_cfg.this_endpoint = my_address;
             sharded<locator::shared_token_metadata> token_metadata;
-            token_metadata.start([] () noexcept { return db::schema_tables::hold_merge_lock(); }, tm_cfg).get();
+            token_metadata.start([] () noexcept { return db::hold_merge_lock(); }, tm_cfg).get();
```

Comment:
Why the namespace change?

Reply:
In a new file I used only db namespace, as I don't see reason for separate nested namespace. But open to suggestions here.

Reply:
It's still schema_tables code, I think it's worthwhile to keep it this way.

Reply:
ok